steps/utilAPI.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+module.exports.getAccessTokenHeader = function() {
```

Comment:
where did you use username/password?

Reply:
From your examples it looks like username/password isn't needed in the header.
You only used the username and passwords in fenceAPI.js: addFileIndices and deleteFileIndices. For seeFileContentEqual you just used the info that I added to the util function.